src/components/Display.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Display(props) {
+  const { result } = props;
+  return (
+    <span className="display">{result}</span>
+  );
+}
```

Comment:
Same as my first comment you could destructure right as you take props as an argument, `function Display ( { result } )`.